src/tsickle.ts

```diff
@@ -2017,6 +2025,41 @@ export function emitWithTsickle(
   };
 }
 
+function addClutzAliases(
+    dtsFileContent: string, sourceFiles: ts.SourceFile[], typeChecker: ts.TypeChecker,
+    host: TsickleHost): string {
+  let reexports = '';
+  for (const sf of sourceFiles!) {
+    const moduleSymbol = typeChecker.getSymbolAtLocation(sf);
+    const moduleExports = moduleSymbol && moduleSymbol.exports || new Map<string, ts.Symbol>();
+
+    for (const e of moduleExports.entries()) {
+      const symbolName = e[0];
+      const symbol = e[1];
+      const clutzSymbolName = 'module$contents$' +
```

Comment:
add a comment that this renaming scheme has to match internal JS compiler. Maybe even point to clutz unit tests.